### PR TITLE
OSPP 2021：final report of wechaty puppet oa

### PR DESCRIPTION
**Pull request by zzzyer**

### CODE REVIEW

**Comment by lijiarui** [comment]
Please DO the following steps first:

1. Get your mentor's approval first.
2. Link back to your ospp 2021 project issue.

@wj-Mcat

**Comment by zzzyer** [comment]
项目地址：https://github.com/wechaty/summer/issues/82

**Comment by lijiarui** [comment]
Please get your mentor's approval first.

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Review by lijiarui — CHANGES_REQUESTED** [review]
It looks strange about your blog file name, please remove `_20211003134518` in the file name.

**Review by huan — CHANGES_REQUESTED** [review]
There are two working in process PR in your project:

- https://github.com/wechaty/wechaty-puppet-official-account/pull/27
- https://github.com/wechaty/wechaty-puppet-official-account/pull/24

Please make sure to pass the CI and get approval from your mentor, by working on them continuously, before you can submit your final project report.

**Comment by huan** [comment]
@wj-Mcat I don't think this PR can be approved because it contains 3 different files under the `_posts/` folder, which is obviously a mistake.

And also, please help our students to get to know how to make their PRs to be merged in the project repo: there's also left lots of problems in her PR, which makes that PR useless.

**Comment by zzzyer** [comment]
> 
> 
> It looks strange about your blog file name, please remove `_20211003134518` in the file name.

Sorry，i have deleted these files.

**Comment by lijiarui** [comment]
> @wj-Mcat I don't think this PR can be approved because it contains 3 different files under the `_posts/` folder, which is obviously a mistake.
> 
> And also, please help our students to get to know how to make their PRs to be merged in the project repo: there's also left lots of problems in her PR, which makes that PR useless.

As @huan said, only when @wj-Mcat re-approves this PR, I will keep review this PR.

**Comment by zzzyer** [comment]
> 
> 
> There are two working in process PR in your project:
> 
>     * [wechaty/wechaty-puppet-official-account#27](https://github.com/wechaty/wechaty-puppet-official-account/pull/27)
> 
>     * [wechaty/wechaty-puppet-official-account#24](https://github.com/wechaty/wechaty-puppet-official-account/pull/24)
> 
> 
> Please make sure to pass the CI and get approval from your mentor, by working on them continuously, before you can submit your final project report.

I have committed a new  PR and passed npm test

**Comment by lijiarui** [comment]
Please get your mentor @wj-Mcat approval as soon as possible.

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Comment by lijiarui** [comment]
@huan How do you think of this PR?

File: jekyll/_posts/2021-10-03-ospp-final-term-wechaty-puppet-oa.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+### PPT展示视频
+
+{% include iframe.html src="https://youtu.be/ySxpkKjv018" %}
+
+Bilibili link: <https://www.bilibili.com/video/BV1344y1t7aU/>
+
+### 项目PPT
+
+{% include iframe.html src="/assets/2021/10-ospp-final-term-wechaty-puppet-oa/final-term-ppt.pdf" %}
```
Comment (lijiarui) [inline jekyll/_posts/2021-10-03-ospp-final-term-wechaty-puppet-oa.md:96]: I think you should add contact info here as template requested: https://github.com/wechaty/summer/blob/master/iscas-open-source-promotion-plan/template/final-term-template.md

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2021-10-03-ospp-final-term-wechaty-puppet-oa.md (hunk 1 above)
Comment (zzzyer) [reply jekyll/_posts/2021-10-03-ospp-final-term-wechaty-puppet-oa.md:96]: Done

Comment (lijiarui) [reply jekyll/_posts/2021-10-03-ospp-final-term-wechaty-puppet-oa.md:96]: Are you sure it is done?

Where is the content info?

Comment (zzzyer) [reply jekyll/_posts/2021-10-03-ospp-final-term-wechaty-puppet-oa.md:96]: I'm sorry, , I had a mistake in understanding before, and now I have added contact info.Is it right？

**Comment by lijiarui** [comment]
@zzzyer Thanks for the update, please get @huan approval, then I will review this PR.

**Comment by lijiarui** [comment]
ping @zzzyer How is this project going on?

**Comment by wj-Mcat** [comment]
ping @huan 

> @zzzyer Thanks for the update, please get @huan approval, then I will review this PR.

**Comment by zzzyer** [comment]
> 
> 
> ping @zzzyer How is this project going on?

I have made some changes and I am waiting for @huan reply

**Comment by huan** [comment]
@zzzyer The two PRs that I have mentioned before were all closed (instead of being merged).

Please update your PR and let me know your updates.

BTW: the master mentor of your project is @wj-Mcat  so I believe the most important approval is from your mentor @wj-Mcat .

@wj-Mcat We should not approval before all required codes/PRs have successfully being merged.

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Review by huan — APPROVED** [review]
LGTM


Could you please link the PR link with your weekly report with a brief introduction?

It will make the report more detail.

Thank you very much for participating in OSPP summer 2021 project, I hope you enjoy it.

Cheers!

**Comment by wj-Mcat** [comment]
ping @zzzyer

**Comment by zzzyer** [comment]
> 
> 
> ping @zzzyer


Thank you for helping me link the issue link.

**Comment by wj-Mcat** [comment]
@lijiarui All changes, including your request changes, have been done, so this pr is ready to be merged. Please help us to merge it.

**Review by zzzyer — COMMENTED** [review]
review changes

**Review by zzzyer — COMMENTED** [review]
delete irrelevant files

**Review by lijiarui — APPROVED** [review]

**Pull request merged**